misc/labels.md

```diff
@@ -59,41 +59,41 @@ This issues is just for discussion or is concerning the build-process, the code-
+This issue was reported as `Problem`, but the issue is reproducible and is now a confirmed bug.
 
 ### `Work In Progress`
-Someone is working on this issue or an PR already exists and needs to be reviewed.<br>
+Someone is working on this issue or a pull request already exists and needs to be reviewed.<br>
 
 ### `Fixed awaiting release`
-This Issues was fixed or implemented in the "develop" branch to mark issues that have been fixed in development but are not released yet and therefore should be still open.<br>
-This issues should be closed if after the
+This Issues is fixed or implemented in the "develop" branch but is not released yet and therefore should be still open.<br>
+This issues should be closed if after the changes are merged into the "master" branch.
 
 ### `For everyone!`
 This is a good issue to start working on if you are new to vis.js and want to help.<br>
 This label is also used for labels that may concern a lot of vis.js users.
 
 ### `IE / Edge`
-This is issues concerns a problem in Microsoft Internet Explorer or the Edge browser.<br>
+These issues concern a problem with the Microsoft Internet Explorer or Edge browser.<br>
 
 ### `Issue Inactive`
-Issues marked as `question` or `problem` get marked as inactive if the author is not responsive or the topic is old.<br>
-If an issues is marked as inactive for about 2 weeks it can be closed without any hesitation.
+Issues marked as `question` or `problem` get marked as inactive when the author is not responsive or the topic is old.<br>
+If an issue is marked as inactive for about 2 weeks it can be closed without any hesitation.
 
 ### `PRIORITY`
-Normally this is used for major Bugs. There should only exist one or two issues marked as PRIORITY at the same time.<br>
-This issues need to be handled before all others.
+In general this is used for major bugs. There should only exist one or two issues marked as PRIORITY at the same time.<br>
+These issues need to be handled before all others.
 
 ### `Requires breaking change`
 A lot of code needs to be changed to implement this. This is maybe something for a major release or something for someone with a lot of time on their hands :-)
 
 ### `invalid`
-This is a non-issue.<br>
-Maybe somebody just created an empty issue, picked the wrong project or something like this.<br>
+This is not a valid issue.<br>
+Someone just created an empty issue, picked the wrong project or something similar.<br>
 This can also be used for pull-request to a non-develop branch oder something similar.<br>
-This issue or PR should be closed immediately.
+This issue or pull request should be closed immediately.
 
 ### `waiting for answer/improvement`
-This is mostly used for pull-requests were a review requested some change and the owner is not answering.
+This is mostly used for pull requests were a reviewer requested some changes and the owner has not responded yet.
```

Comment:
At the moment we just leave these PRs open. Maybe somebody will pick them up at some point!?